### PR TITLE
new post for new website

### PR DESCRIPTION
**Pull request by titusfortner**
@bret @p0deje, et al, does this look ok? What information should be added/removed?

### CODE REVIEW

**Comment by p0deje** [comment]
:+1:

File: _contributors/rodionov_alex.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Alex Rodionov
+twitter: p0deje
+location: Omsk, Russia
+website: http://watirtight.com
```
Comment (Rodney-QAGeek) [inline _contributors/rodionov_alex.md]: @titusfortner is this right?

Comment (titusfortner) [inline _contributors/rodionov_alex.md]: Already fixed: https://github.com/watir/watir.github.io/commit/c80c2e

**Comment by titusfortner** [comment]
This content has already been merged

**Pull request closed without merging**